cxplat/src/cxplat_winkernel/cxplat_winkernel.vcxproj

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: MIT
+-->
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We can do this in a follow up PR, but this project file needs some work. It won't load on xbox for instance.

Reply:
Filed #85 to track